source/agora/common/crypto/ECC.d

```diff
@@ -20,6 +20,7 @@ module agora.common.crypto.ECC;
 import agora.common.Hash;
 import agora.common.Serializer;
 import agora.common.Types;
+import agora.utils.PrettyPrinter;
```

Comment:
This is your problem. And a big no.

Reply:
What did @Geod24 tell you yesterday? Always blame drey.